tencentcloud/resource_tc_lighthouse_instance_test.go

```diff
@@ -63,6 +63,12 @@ func TestAccTencentCloudLighthouseInstanceResource_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("tencentcloud_lighthouse_instance.instance", "renew_flag", "NOTIFY_AND_MANUAL_RENEW"),
 				),
 			},
+			{
+				ResourceName: "tencentcloud_lighthouse_instance.instance",
```

Comment:
Since Lighthouse instances currently do not support pay-as-you-go, I am unable to initiate the acceptance tests. This commit will include the content for Lighthouse instance import acceptance tests. Could a developer with testing permissions please run the tests? Thank you.